components/infobox/wikis/pubgmobile/infobox_map_custom.lua

```diff
@@ -0,0 +1,131 @@
+---
+-- @Liquipedia
+-- wiki=pubgmobile
+-- page=Module:Infobox/Map/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+local Template = require('Module:Template')
+
+local Injector = Lua.import('Module:Infobox/Widget/Injector', {requireDevIfEnabled = true})
+local Map = Lua.import('Module:Infobox/Map', {requireDevIfEnabled = true})
+
+local Abbreviation = require('Module:Abbreviation')
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+
+local CustomMap = Class.new()
+
+local CustomInjector = Class.new(Injector)
+
+local _args
+
+local GAME = {
+	mobile = '[[Mobile]]',
+	newstate = '[[New State]]',
+	peace = '[[Peacekeeper Elite|Peace Elite]]',
+	bgmi = '[[Battlegrounds Mobile India|BGMI]]',
+}
+
+local MODES = {
+	solo = '[[:Category:Solos Mode Tournaments|Solos]]',
+	duo = '[[:Category:Duos Mode Tournaments|Duos]]',
+	squad = '[[:Category:Squads Mode Tournaments|Squads]]',
+	['2v2'] = '[[:Category:2v2 TDM Tournaments|2v2 TDM]]',
+	['4v4'] = '[[:Category:4v4 TDM Tournaments|4v4 TDM]]',
+	['war mode'] = '[[:Category:War Mode Tournaments|War Mode]]',
+	default = '',
+}
+MODES.solos = MODES.solo
+MODES.duos = MODES.duo
+MODES.squads = MODES.squad
+MODES.tdm = MODES['2v2']
```

Comment:
Can a map only have one mode? I haven't played PUBGM, but that doesn't seem correct to me?

Reply:
I assumed it was similar to apex in that maps did have one mode. A cursory look showed a couple maps used for a small number of TDM tournaments that didnt have pages yet. 
I'll add support for multiple entries

Reply:
For maps these are a bit different, 

In PUBGM and PUBG The maps are not dictate by the perspective but define as the game mode itself

For PUBGM this splits into
- Battle Royale Maps (called **Classic** in the game)
- Arena Maps (often called **Unranked** or **Arena** in the game contains things like either TDM or other non-BR/TDM game modes)

Arena one for PUBGM does include maps that is just a portion of the Battle Royale Maps
and also maps that is entirely separate and create just for the mode (not a chunk of any BR Maps) so I'd say the modes here may still be working 

just need to change up to
- add support entry for **Battle Royale**
- add support entry for **Arena**
- remove all else

Reply:
This was going to be my suggestion, that's how Apex has it set up
Battle Royale / Arenas